docs/configuration/configuration.md

```diff
@@ -35,4 +34,29 @@ To use an existing FeatureFlagConfiguration custom resource, start flagD with th
 
 ```shell
 flagd start --uri core.openfeature.dev/default/my_example
+```
+
+### Source Configuration
+
+While a URI may be passed to flagd via the `--uri` flag, some implementations may require further configurations. In these cases the `--sources` flag should be used.
+The flag takes a string argument, which should be a JSON representation of an array of `ProviderConfig` objects. Alternatively, these configurations should be passed to
+flagd via config file, specified using the `--config` flag.
+
+| Field       | Type  | 
+|------------|------------------------------------|
+| uri | required `string` |  |
+| provider   | required `string` (`file`, `kubernetes`, `http` or `grpc`) |
+| bearerToken     | optional `string`        |
+
+The `uri` field values do not need to follow the [URI patterns](#uri-patterns), the provider type is instead derived from the provider field.
```

Comment:
What would happen if the `file` uri prefix was included? Can you give more examples below? I'm particularly interested in how the HTTP source config should look but including one for gRPC and Kubernetes would be useful as well.

Reply:
Added logic to parse `file:` and `core.openfeature.dev` prefixes, and added a reference to this in the docs